### PR TITLE
feature/add-entry

### PR DESCRIPTION
**Pull request by Marpop**

### CODE REVIEW

File: logs/views.py [hunk 1]
```diff
@@ -1,14 +1,52 @@
...
+
+    contex = {'form': form}
+    return render(request, 'logs/add_entry.html', contex)
+
+
+@login_required
+def edit_entry(request, entry_id):
+    entry = Entry.objects.get(id=entry_id)
+    if entry.owner != request.user:
+        raise Http404
```
Comment (piotermiarer) [inline logs/views.py]: This page exists but is forbidden so 403 error could be more suitable.

`from django.http import HttpResponseForbidden`
`raise HttpResponseForbidden()`

Comment (Marpop) [reply logs/views.py]: Oh, thanks 👍

Comment (Marpop) [reply logs/views.py]: @piotermiarer why not 401 instead of 403?

File: logs/templates/logs/edit_entry.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends "logs/base.html" %}
+
+{% block content %}
+
+    <p>Edit entry:</p>
+
+    <form method='post' action="{% url 'logs:edit_entry' entry.id %}">
```
Comment (k4ju) [inline logs/templates/logs/edit_entry.html]: I don't see any styling here. If you're doing an entire app and not an API, you should deliver whole feature — there are views, there should be some CSS

File: logs/forms.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from django import forms
+
+from .models import Entry
+
```
[the remark this answers is not in the document]
Comment (Marpop) [reply logs/forms.py]: Yep, pycodestyle -> E302 expected 2 blank lines, found 1

File: logs/migrations/0003_entry_owner.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2017-10-10 13:50
```
[the remark this answers is not in the document]
Comment (Marpop) [reply logs/migrations/0003_entry_owner.py]: @piotermiarer xD

File: logs/models.py [hunk 5]
```diff
@@ -1,9 +1,11 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 
 class Entry(models.Model):
     text = models.TextField()
     date_added = models.DateTimeField(auto_now_add=True)
+    owner = models.ForeignKey(User)
```
[the remark this answers is not in the document]
Comment (Marpop) [reply logs/models.py]: maybe it's better :)

File: logs/templates/logs/add_entry.html [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{% extends "logs/base.html" %}
+
+{% block content %}
+
+    <p>New entry:</p>
```
[the remark this answers is not in the document]
Comment (Marpop) [reply logs/templates/logs/add_entry.html]: I know, I have it on my 'to do' list

File: logs/templates/logs/edit_entry.html (hunk 2 above)
Comment (Marpop) [reply logs/templates/logs/edit_entry.html]: See next PR :)

File: logs/models.py [hunk 7]
```diff
@@ -1,9 +1,11 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 
 class Entry(models.Model):
     text = models.TextField()
     date_added = models.DateTimeField(auto_now_add=True)
```
[the remark this answers is not in the document]
Comment (Marpop) [reply logs/models.py]: IMO it's not necessary, but I will think about it. Thanks

[the remark this answers is not in the document]
Comment (k4ju) [reply logs/models.py]: If you don't keep info you loose potential value. If in doubt, always better to more than to regret later

[the remark this answers is not in the document]
Comment (Marpop) [reply logs/models.py]: Roger, agree

**Pull request merged**